infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/project/OpenShiftWorkspaceServiceAccount.java

```diff
@@ -97,6 +99,17 @@ private void createExecRole(OpenShiftClient osClient, String name) {
     osClient.roles().inNamespace(projectName).create(execRole);
   }
 
+  private void createViewRole(OpenShiftClient osClient, String name) {
```

Comment:
Should not the same role be created in `KubernetesWorkspaceServiceAccount` class?

Reply:
Right. Missed that since the motivation for this change was an OpenShift use case from a customer.